drivers/bridge/bridge.go

```diff
@@ -145,6 +146,159 @@ func (c *NetworkConfiguration) Validate() error {
 	return nil
 }
 
+// FromMap retrieve the configuration data from the map form.
+func (c *networkConfiguration) FromMap(data map[string]interface{}) error {
```

Comment:
@mrjana, it just does not work to rely on generateFromModel on data that was json encoded/decoded when you have complex types. I lost so much time on this. 
I looked at the changes in docker daemon. They are minimal.